syntax/sql/sqlx_base.vim

```diff
@@ -0,0 +1,108 @@
+" Vim syntax file
+" Language: SQLX
+" Maintainer: Andres Lowrie
+" Repository: https://github.com/andres-lowrie/vim-sqlx
+" License: VIM
+" Credits: https://github.com/vim/vim/blob/master/runtime/syntax/sqloracle.vim
+
+
```

Comment:
lmk if you want me to remove this, as mentioned I just copied/pasta 'ed the other code

Reply:
Hey @andres-lowrie  for me its fine! perhaps just change Repository docs for current one?